app/models/vmdb_database/seeding.rb

```diff
@@ -38,19 +49,63 @@ def db_disk_size(disk)
       return nil if err.message.include?("does not exist")
       raise
     end
-  end
 
-  def seed
-    mine = evm_tables.includes(:text_tables, :vmdb_indexes).index_by(&:name)
+    def seed_tables
+      db = my_database
```

Comment:
> Being in a transaction doesn't affect re-query

I am less concerned about "re-query" and more concerned about when it is `.new`.  I have seen (in tests), when I was trying to access something created in a different process, that the record wouldn't be accessible.

But, that might have been because it then made the query on a different PG connection that caused the issue.  Does that sound correct?  If so, then I guess this is fine.